README.md

```diff
@@ -103,7 +103,29 @@ Workmanager().registerOneOffTask(
 );
 ```
 
-Tasks registered this way will appear in the callback dispatcher using as `Workmanager.iOSBackgroundProcessingTask`.
+**NOTE**
+
+- We have to register in ApplicationDelegate class as below example
+
+```swift
+WorkmanagerPlugin.registerTask(withIdentifier: "task-identifier")
+```
+
+- We must also define your `task-identifier` in `info.plist` file
+
+```
+<dict>
+  ....
+	<key>BGTaskSchedulerPermittedIdentifiers</key>
+	<array>
+		<string>workmanager.background.task</string>
```

Comment:
This one is not strictly needed anymore, right?

Reply:
Yeah, right, it is not strictly anymore. I just kept there as an example to follow.